knowledge-base/dropdowns-show-loading-indicator-skeletons-during-virtual-scrolling.md

```diff
@@ -0,0 +1,162 @@
+---
+title: Display Loading Indicators during Virtual Scrolling
+description: Learn how to display loading skeleton indicators during virtual scrolling in a Telerik Blazor AutoComplete, ComboBox, DropDownList, MultiColumnComboBox, and MultiSelect componenta.
+type: how-to
+page_title: How to Display Loading Indicators during Virtual Scrolling
+slug: dropdowns-kb-show-loading-indicator-skeletons-during-virtual-scrolling
+tags: telerik, blazor, virtualization, autocomplete, combobox, dropdownlist, multicolumncombobox, multiselect
+ticketid: 1587267, 1699463
+res_type: kb
+---
+
+## Environment
+
+<table>
+    <tbody>
+        <tr>
+            <td>Product</td>
+            <td>
+                AutoComplete for Blazor, <br />
+                ComboBox for Blazor, <br />
+                DropDownList for Blazor, <br />
+                MultiColumnComboBox for Blazor, <br />
+                MultiSelect for Blazor
+            </td>
+        </tr>
+    </tbody>
+</table>
+
+## Description
+
+This KB article answers the following questions:
+
+* How to show loading indicators in the Telerik select components during virtual scrolling?
```

Comment:
Just checking if "select components" is a valid term. We're usually calling these dropdown components, or you might call them "selection components" (but this is not widely used and might not be instantly recognizable).